lib/tasks/unsubscribe.rake

```diff
@@ -0,0 +1,26 @@
+require 'csv'
+
+namespace :unsubscribe do
+  def unsubscribe!(email_address)
+    subscriber = Subscriber.find_by(address: email_address)
+    if subscriber.nil?
+      puts "Subscriber #{email_address} not found"
+    else
+      puts "Unsubscribing #{email_address}"
+      UnsubscribeService.subscriber!(subscriber, :unsubscribed) unless subscriber.nil?
```

Comment:
We don't need the extra `unless ...` here.

Reply:
Yeah I saw that as you were reviewing, a remnant of an earlier approach!